LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Felix Krause
+Copyright (c) 2015-present the fastlane Authors
```

Comment:
Was following legal's spelling "the Fastlane Authors", but then lower cased "fastlane" for branding reasons.